manager/dispatcher/dispatcher.go

```diff
@@ -213,8 +216,15 @@ func (d *Dispatcher) Run(ctx context.Context) error {
 		case v := <-configWatcher:
 			cluster := v.(state.EventUpdateCluster)
 			d.mu.Lock()
-			d.config.HeartbeatPeriod = time.Duration(cluster.Cluster.Spec.Dispatcher.HeartbeatPeriod)
-			d.nodes.updatePeriod(d.config.HeartbeatPeriod, d.config.HeartbeatEpsilon, d.config.GracePeriodMultiplier)
```

Comment:
I think there is a bug in the old code: if we update cluster without specifying value of `cluster.Cluster.Spec.Dispatcher.HeartbeatPeriod`, default value 0 will be used. This may lead to some bad results.